addons/udes_delivery_control/models/stock_picking.py

```diff
@@ -284,6 +246,14 @@ def write(self, values):
 
     @api.model
     def create(self, values):
+        picking_type_dc = self.env.ref("udes_delivery_control.picking_type_delivery_control")
+        picking_type_id = values.get("picking_type_id")
+        if picking_type_id and picking_type_id == picking_type_dc.id:
+            # Set loading type to unload if picking is of delivery control type
+            # Note: u_loading_type is of related type field and compute attribute
+            # is not compatible/called. Hence, we are setting it's default value in create method
+            values["u_loading_type"] = "unload"
```

Comment:
Could we just do this as default value for DC ?

Reply:
I tried all approaches but none seems to work
1. set default on fields - Using this method we have no way to get the value of picking type unless we set the context from where we create picking. 
2. Override default_get method - When the values are set in this method it gets changed by odoo somewhere as the field u_loading_type is a related field
3. compute on-field - Does not get called at all. Might be something to do with a related type of field

Reply:
Fair enough, worthed asking though :)